src/charm.py

```diff
@@ -193,15 +205,25 @@ def _get_hydra_endpoint_info(self) -> str:
                 logger.info("No hydra-endpoint-info relation data found")
         return hydra_url
 
-    def _get_kratos_endpoint_info(self, endpoint_type: str) -> str:
-        kratos_url = ""
-        if self.model.relations[KRATOS_ENDPOINTS_INTEGRATION_NAME]:
+    def _get_kratos_info(self, key: str) -> str:
+        kratos_key = ""
+        if self.kratos_info.is_ready():
+            try:
+                kratos_info = self.kratos_info.get_kratos_info()
+                kratos_key = kratos_info[key]
```

Comment:
Just curious why we name the value in the integration databag as `xxx_key`.

Reply:
I got rid of that naming